src/common/model/DealState.ts

```diff
@@ -12,7 +12,7 @@ export default interface DealState {
   duration: number,
   price: number, // unit is Fil
   verified: boolean,
-  state: 'proposed' | 'published' | 'active' | 'slashed' | 'error',
+  state: 'proposed' | 'published' | 'active' | 'expired' | 'slashed' | 'error',
```

Comment:
I think we can name it proposal_expired to differentiate from actual deal expiration